packages/core/src/codewhisperer/commands/types.ts

```diff
@@ -11,6 +12,12 @@ export const cwTreeNodeSource = 'codewhispererTreeNode'
 export const cwQuickPickSource = 'codewhispererQuickPick'
 /** Indicates a CodeWhisperer command was executed through the Amazon Q chat pane */
 export const amazonQChatSource = 'amazonQChat'
+/** Indicates a CodeWhisperer command was executed during the first start of the extension */
+export const firstStartUpSource = ExtStartUpSources.firstStartUp
+/** Indicates a CodeWhisperer command was executed as a result of signing out */
+export const cwSignOut = 'codewhispererSignOut'
+/** Indicates a CodeWhisperer command was executed as a result of selecting an ellipses menu item */
+export const cwEllipsesMenu = 'ellipsesMenu'
```

Comment:
Why are these extra variables needed? Using literals works the same (including IDE completion, goto definition, etc), and avoids this extra boilerplate.